features/support/time_machine_helpers.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module TimeMachineHelpers
+  def time_machine(till)
+    travel_to(Time.zone.now)
+
+    while Time.zone.now < till
+      travel_to(1.day.from_now)
+      run_jobs
+    end
+    travel_to(till)
+  rescue StandardError
+    travel_back
+    raise
+  end
+
+  private
+
+  def run_jobs
+    run(ThreeScale::Jobs::MONTH) if Time.zone.now == Time.zone.now.beginning_of_month
```

Comment:
I think we can make Codeclimate happy and add `now = Time.zone.now` in the beginning of the method

Reply:
Yes, it should be safe since `Time.now` is frozen when this method is called, so it shouldn't alter the behavior.